content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -349,3 +350,106 @@ rules to reach the proper Istio service in the main cluster.
+### mTLS between application pods
+
+The steps to enable mTLS for all application pods are as follows:
+
+1.  Istio control plane cluster deployed with
+    1.  Global mTLS enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+
+1.  Istio remote clusters deployed with
+    1.  Global mTLS enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+        1.  The CA certificate for the remote clusters needs to be signed by the same CA or root CA as the main cluster.
+
+**NOTE:**  The CA certificate steps are identical for both control plane security and application pod security steps.
+
+### Example deployment
+
+The following is an example procedure to install Istio with both control plane mTLS and application pod
+mTLS enabled.  The example sets up a remote cluster with a selector-less service and endpoint for `istio-pilot` to
+allow the remote sidecars to resolve `istio-pilot.istio-system` hostname via its local kubernetes DNS.
+
+1.  *Primary Cluster.*  Deployment of the Istio control plane cluster
+
+    1.  Create the `cacerts` secret from the Istio samples certificate in the `istio-system` namespace:
+
+        {{< text bash >}}
+        $ kubectl create ns istio-system
+        $ kubectl create secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem --from-file=samples/certs/cert-chain.pem
+        {{< /text >}}
+
+    1.  Deploy the Istio control plane with control plane and application pod security enabled
+
+        {{< text bash >}}
+        $ helm template --namespace=istio-system \
+          --values install/kubernetes/helm/istio/values.yaml \
+          --set global.mtls.enabled=true \
+          --set security.selfSigned=false \
+          --set global.controlPlaneSecurityEnabled=true \
+          install/kubernetes/helm/istio > ${HOME}/istio-auth.yaml
+        $ kubectl create -f ${HOME}/istio-auth.yaml
+        {{< /text >}}
+
+1.  *Remote Cluster.*  Deployment of remote cluster's istio components
+
+    1.  Create the `cacerts` secret from the Istio samples certificate in the `istio-system` namespace:
+
+        {{< text bash >}}
+        $ kubectl create ns istio-system
+        $ kubectl create secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem --from-file=samples/certs/cert-chain.pem
+        {{< /text >}}
+
+    1.  Set endpoint IP environment variables as in the [above section](#set-environment-variables-for-pod-ips-from-istio-control-plane-needed-by-remote)
```

Comment:
"as in the above section" "As described in [setting environment variables].

Reply:
will fix